test/cadet_web/controllers/auth_controller_test.exs

```diff
@@ -26,7 +25,7 @@ defmodule CadetWeb.AuthControllerTest do
   alias Cadet.Auth.Guardian
   alias CadetWeb.AuthController
 
-  @token if System.get_env("TOKEN"), do: System.get_env("TOKEN"), else: "token"
+  @code System.get_env("CODE") || "code"
```

Comment:
Do you really need the `|| "code"`? What if you just remove that and let `@code` be `nil`